test/tree.test.ts

```diff
@@ -125,15 +125,17 @@ test.each([
       "kiegroup/kie-uberfire-extensions",
       "kiegroup/droolsjbpm-knowledge",
       "kiegroup/drools",
-      "kiegroup/optaplanner",
+      // not sure if this is really dependent on Appformer
+      //"kiegroup/optaplanner",
       "kiegroup/jbpm",
       "kiegroup/droolsjbpm-integration",
       "kiegroup/kie-wb-playground",
       "kiegroup/kie-wb-common",
       "kiegroup/drools-wb",
       "kiegroup/jbpm-work-items",
       "kiegroup/jbpm-wb",
-      "kiegroup/optaplanner-wb",
```

Comment:
i found the removal of optaplanner on project dependencies definitions so it is expected now to be green.